src/skin/adminhtml/default/default/omise/omise-dashboard.js

```diff
@@ -0,0 +1,523 @@
+                }
+
+                jQuery('.charge-loading.load-background').hide();
+                chargeData = charge;
+                if(callback) callback();    
+            });
+        }
+
+        // handle charge pagination
+        var nextChargePage = function(direction){
+            np = parseInt(jQuery('#charge-pn').text()) + direction;
+            np = np < 1 ? 1 : np;
+            np = np > charge_total ? charge_total : np;
+            loadChageTable(np, function(){
+
+                jQuery('#charge-btn-back').show();
+                if(np == 1) jQuery('#charge-btn-back').hide();
+
+                jQuery('#charge-btn-first').show();
+                if(np == 1) jQuery('#charge-btn-first').hide();
+
+                 jQuery('#charge-btn-next').show();
+                if(np == charge_total) jQuery('#charge-btn-next').hide();
+
+                jQuery('#charge-btn-last').show();
+                if(np == charge_total) jQuery('#charge-btn-last').hide();
+
+              jQuery('#charge-pn').text(np);  
+            });
+        }
+
+        var gotoChargeFirstPage = function(){
+            loadChageTable(1, function(){
+                jQuery('#charge-btn-last').show();
+
+                if(charge_total<=1){
+                    jQuery('#charge-btn-back').hide();
+                    jQuery('#charge-btn-first').hide();
+                    jQuery('#charge-btn-next').hide();
+                    jQuery('#charge-btn-last').hide();
+                }else{
+                    jQuery('#charge-btn-back').hide();
+                    jQuery('#charge-btn-first').hide();
+                    jQuery('#charge-btn-next').show();
+                }
+
+              jQuery('#charge-pn').text(1);  
+            });
+        }
+
+        var gotoChargeLastPage = function(){
+            loadChageTable(charge_total, function(){
+
+                jQuery('#charge-btn-back').show();
+                jQuery('#charge-btn-first').show();
+                jQuery('#charge-btn-next').hide();
+                jQuery('#charge-btn-last').hide();
+
+              jQuery('#charge-pn').text(charge_total);  
+            });
+        }
+
+        // first load chrage table for the first page
+        gotoChargeFirstPage();
+
+        var setTransferTable = function(i, data){
+            var tr = '<tr>';
+
+            if(data){
+                tr += '<td>฿' + data.amount + '</td>';
+
+                tr += '<td>'+data.id+'</td>';
+                tr += '<td>'+(data.failure_code?'<span class="error-label">Fail</span>':data.sent?data.paid?'<span class="success-label">Paid</span>':'<span class="primary-label">Request sent</span>':'<span class="warning-label">Requesting</span>')+'</td>';
+                tr += '<td>'+(data.failure_code?('('+data.failure_code+')'+data.failure_code):'-')+'</td>';
+                tr += '<td>'+data.created+'</td>';
+
+                var td5 = jQuery('<td>');
+                td5.addClass('a-center');
```

Comment:
@duzydook Do we still need this line after you change your code to concatenate string style ? what if we change this line to `tr += '<td class="a-center">';`